examples/Load/Program.fs

```diff
@@ -9,16 +9,16 @@ open Suave.Http.Applicatives
 open Suave.Http.Files
 open Suave.Logging
 
-let logger = Loggers.sane_defaults_for LogLevel.Verbose
+let logger = Loggers.saneDefaultsFor LogLevel.Verbose
```

Comment:
When we're changing things, are we happy with this name? I find it a bit 'shoot from the hip'...